client/src/app/unauthorized.component.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { describe, expect, it } from 'vitest';
+
+import { render, screen, userEvent } from '~/test/utils';
+
+import { Unauthorized } from './unauthorized.component';
+
+describe('Unauthorized', () => {
+  it('should render unauthorized component', () => {
+    render(<Unauthorized />);
+
+    expect(screen.getByRole('heading', { name: /unauthorized/i })).toBeInTheDocument();
+    expect(screen.getByText(/you do not have access to the requested page./i)).toBeInTheDocument();
+    expect(screen.getByRole('button', { name: /go back/i })).toBeInTheDocument();
+  });
+
+  it('should move one step backward from unauthorized page when clicks on the "Go Back" button', async () => {
+    const user = userEvent.setup();
```

Comment:
I'm not familiar with this use of `userEvent`, what is this doing?

Reply:
There is a link to the documentation which explains that: https://testing-library.com/docs/user-event/intro/ Mark  just asked me to do it in that way.

Reply:
Ok cool, good to know